train.py

```diff
@@ -73,12 +75,23 @@ def generate_subspectrogram(duration_s=config.SUBSPECTROGRAM_DURATION_S, fft_rat
         checkpoint.epoch.assign_add(1)
         checkpoint_manager.save()
 
-    print("======================== evaluation on validation data =========================")
-    # test model on validation set
-    utils.test_model(model, val_spectrograms, val_labels, val_loss, val_accuracy)
+        if epoch % 10 == 0 or epoch == config.NB_EPOCHS - 1:
```

Comment:
The epoch period (10 here) could be a config variable

Reply:
True